formatTest/unit_tests/expected_output/basicStructures.re

```diff
@@ -643,10 +643,12 @@ let anotherRecord = {
 
 let anotherRecord = {
   ...
-    SomeReally.longFunctionCall{
-      passingRecordField: 0,
-      andThisOtherRecordField: 10
-    },
+    SomeReally.longFunctionCall(
```

Comment:
Is it possible to have this formatted to: 

```reason
foo({
  a: 1
})
```

We can do that in another diff. Just wondering if it's easy.

Reply:
Not super easy, probably medium difficulty.
Refmt has a 'unparsing' algorithm which needs to be modified a bit. In essence that would allow us the stick `({` together.